src/m_imp.h

```diff
@@ -93,7 +93,8 @@ void pd_globalunlock(void);
 /* misc */
 #define SYMTABHASHSIZE 1024
 
-EXTERN t_pd *glob_evalfile(t_pd *ignore, t_symbol *name, t_symbol *dir);
+EXTERN t_pd* glob_evalfile(t_pd* ignore, t_symbol* name, t_symbol* dir);
+EXTERN t_pd* glob_evaltext(const char* data, size_t length, t_symbol* name, t_symbol* dir);
```

Comment:
why are those functions declared both in `m_imp.h` and `m_pd.h`?

Reply:
yes, they should only be exported in a single header. do you think it's ok to put/move them to `m_pd.h`?

Reply:
i thought having providing a public API was the entire point of the exercise.
since `m_imp.h` is by definition the *private* API, `m_pd.h` looks like the correct place.

Reply:
just asking because libpd uses `m_imp.h`, so it doesn't *need* to be in `m_pd.h`, but for external authors it's of course nicer to have it in the public header.

BTW, the signature of `glob_evalfile` is really ugly, but we can't change it because it's exported, I guess...